librz/core/cmd_descs/cmd_debug.yaml

```diff
@@ -357,33 +357,134 @@ commands:
             args:
               - name: flag
                 type: RZ_CMD_ARG_TYPE_FLAG
-  - name: dts
-    summary: Debug trace session commands
+  - name: dt
+    summary: Trace commands
     subcommands:
-      - name: dts+
-        summary: Start trace session
-        cname: cmd_debug_start_trace_session
-        args: []
-      - name: dts-
-        summary: Stop trace session
-        cname: cmd_debug_stop_trace_session
+      - name: dt
+        summary: Get trace info at address
+        cname: cmd_debug_trace
+        args:
+          - name: addr
+            type: RZ_CMD_ARG_TYPE_RZNUM
+      - name: dtl
+        summary: List all traces
+        cname: cmd_debug_traces
         args: []
-      - name: dtst
-        cname: cmd_debug_save_trace_session
-        summary: Save trace sessions to disk
+        modes:
+          - RZ_OUTPUT_MODE_QUIET
+          - RZ_OUTPUT_MODE_STANDARD
+          - RZ_OUTPUT_MODE_RIZIN
+          - RZ_OUTPUT_MODE_TABLE
+      - name: dt+
+        summary: Add trace for address N times
+        cname: cmd_debug_trace_add
         args:
-          - name: dir
-            type: RZ_CMD_ARG_TYPE_FILE
-      - name: dtsf
-        cname: cmd_debug_load_trace_session
-        summary: Load trace sessions to disk
+          - name: addr
+            type: RZ_CMD_ARG_TYPE_RZNUM
+          - name: times
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
+      - name: dt++
+        summary: Add trace for some address
+        cname: cmd_debug_trace_add_addrs
         args:
-          - name: dir
-            type: RZ_CMD_ARG_TYPE_FILE
-      - name: dtsm
-        cname: cmd_debug_list_trace_session_mmap
-        summary: List current memory map and hash
+          - name: addrs
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            flags: RZ_CMD_ARG_FLAG_ARRAY
+            optional: true
```

Comment:
I would say that at least one should be passed as argument, so i would remove optional here.